manim/mobject/three_dimensions.py

```diff
@@ -254,6 +260,59 @@ def param_surface(u, v):
 # Specific shapes
 
 
+class Plane(ParametricSurface):
+    """
+    A 3D Plane Mobject.
+
+    Examples
+    --------
+
+    .. manim:: PlaneExample
+        :save_last_frame:
+
+        class PlaneExample(ThreeDScene):
+            def construct(self):
+                self.set_camera_orientation(PI / 3, 100 * DEGREES)
+                ax = ThreeDAxes()
+                normal = [1, 1, 1]
+                normal_vector = Arrow3D(ORIGIN, normal, color=RED)
+                dot = Dot3D([-2, -1, 1], color=ORANGE)
+                plane = Plane(point=dot.get_center(), normal_vect=normal)
+                self.add(ax, normal_vector, dot, plane)
+                self.wait()
+
+    Parameters
+    ----------
+    point: :class:`numpy.ndarray`
+        A point that lies on the plane.
+    normal_vect: :class:`numpy.ndarray`
+        A vector perpendicular to the plane.
+    kwargs
+        To be passed to :class:`ParametricSurface`
+    """
+
+    def __init__(self, point=ORIGIN, normal_vect=[1, 1, 1], **kwargs):
+        self.normal_vect = np.array(normal_vect)
+        u_range = kwargs.pop("u_range", [-2 * PI, 2 * PI])
+        v_range = kwargs.pop("v_range", [-2 * PI, 2 * PI])
+        super().__init__(
+            lambda u, v: np.array([u, v, 0]), u_range=u_range, v_range=v_range, **kwargs
+        )
+        spherical = cartesian_to_spherical(normal_vect)
+        axis = [np.sin(spherical[1]), -np.cos(spherical[1]), 0]
+        self.rotate(-spherical[2], axis=axis)
+        self.shift(projected_vector(self.normal_vect, point))
+
+    def line_perp_to_plane(self, point, **kwargs):
```

Comment:
typing + a return type (can be specifed by adding a `->` before the colon), Plus, I think the length of the line should be able to be adjusted here.